src/librustc_metadata/diagnostics.rs

```diff
@@ -91,6 +91,204 @@ You need to link your code to the relevant crate in order to be able to use it
 well, and you link to them the same way.
 "##,
 
+E0466: r##"
+Invalid macro import declarations.
```

Comment:
You're supposed to be a little more exhaustive on the error, not just repeat the error message.